Test/Web/FunctionalTests/TestApps/WebAppFW45/ApplicationInsights.config

```diff
@@ -43,6 +43,7 @@
       </Handlers>
     </Add>
     <Add Type="Microsoft.ApplicationInsights.Web.ExceptionTrackingTelemetryModule, Microsoft.AI.Web"/>
+    <Add Type="Microsoft.ApplicationInsights.Web.AspNetDiagnosticTelemetryModule, Microsoft.AI.Web"/>
```

Comment:
>AspNetDiagnosticTelemetryModule [](start = 49, length = 31)

so we need both - `RequestTrackingTelemetryModule` and `AspNetDiagnosticTelemetryModule`?